meta/store.go

```diff
@@ -1705,6 +1746,33 @@ func (fsm *storeFSM) Apply(l *raft.Log) interface{} {
 	return err
 }
 
+func (fsm *storeFSM) applyRemovePeerCommand(cmd *internal.Command) interface{} {
+	ext, _ := proto.GetExtension(cmd, internal.E_RemovePeerCommand_Command)
+	v := ext.(*internal.RemovePeerCommand)
+
+	id := v.GetID()
+	addr := v.GetAddr()
+
+	// Only do this if you are the leader
+	if fsm.raftState.isLeader() {
+		//Remove that node from the peer
```

Comment:
Not sure what causes this.  It's fine locally, and go fmt doesn't see it either.  I even opened it in a couple of other editors to be sure as well.  Apparently only github sees the extra space?  I've seen this before as well and I'm still not sure what is going on.